src/pages/downloads.tsx

```diff
@@ -0,0 +1,225 @@
+                  Android library archives and iOS XCode frameworks
+                </Text>
+              </ListItem>
+            </UnorderedList>
+
+            <Text textStyle='quick-link-text'>
+              Please select your desired platform from the lists below and download your bundle of choice. Please be aware that the MD5 checksums are provided by our binary hosting platform (Azure Blobstore) to help check for download errors. For security guarantees please verify any downloads via the attached PGP signature files (see{' '}
+              <Link
+                href={'#pgpsignatures'}
+                variant='light'
+              >
+                OpenPGP
+              </Link>{' '}
+              Signatures for details).
+            </Text>
+          </Stack>
+        </DownloadsSection>
+
+        <DownloadsSection sectionTitle='Stable releases' id='stablereleases'>
+          <Stack p={4} borderBottom='2px solid' borderColor='brand.light.primary'>
+            <Text textStyle='quick-link-text'>
+              These are the current and previous stable releases of go-ethereum, updated automatically when a new version is tagged in our{' '}
+              <Link
+                href={GETH_REPO_URL}
+                isExternal
+                variant='light'
+              >
+                GitHub repository.
+              </Link>
+            </Text>
+          </Stack>
+
+          {/* TODO: swap test data for real data */}
+          <DownloadsTable data={testDownloadData.slice(0, amountStableReleases)}/>
+
+          <Stack sx={{ mt: '0 !important' }}>
+            <Link as='button' variant='button-link-secondary' onClick={() => {
+              updateAmountStables(amountStableReleases+10)
+            }}>
+              <Text
+                fontFamily='"JetBrains Mono", monospace'
+                fontWeight={700}
+                textTransform='uppercase'
+                textAlign='center'
+                p={4}
+              >
+                Show older releases
+              </Text>
+            </Link>
+          </Stack>
+        </DownloadsSection>
+
+        <DownloadsSection sectionTitle='Develop builds' id='developbuilds'>
+          <Stack p={4} borderBottom='2px solid' borderColor='brand.light.primary'>
+            <Text textStyle='quick-link-text'>
+              These are the develop snapshots of go-ethereum, updated automatically when a new commit is pushed into our{' '}
+              <Link
+                href={GETH_REPO_URL}
+                isExternal
+                variant='light'
+              >
+                GitHub repository.
+              </Link>
+            </Text>
+          </Stack>
+
+         {/* TODO: swap for real data */}
+          <DownloadsTable data={testDownloadData.slice(0, amountDevelopBuilds)} />
+
+          <Stack sx={{ mt: '0 !important' }}>
+            <Link as='button' variant='button-link-secondary' onClick={() => {
```

Comment:
onClick method should be moved to a named function for more clarity, something like

```
onClick={showMoreDevelopBuilds}
```